projects/plugins/crm/modules/woo-sync/includes/class-woo-sync-background-sync-job.php

```diff
@@ -1705,6 +1725,39 @@ public function woocommerce_order_to_crm_objects(
 
 			}
 
+			// --- Process any present fee in the order --- //
+			$fees = $order->get_fees();
+
+			if ( is_array( $fees ) && count( $fees ) > 0 ) {
+				foreach ( $fees as $fee ) {
+					if ( $fee instanceof \WC_Order_Item_Fee ) {
+						$new_line_item = array(
+							'order'    => $order_post_id, // passed as parameter to this function
+							'currency' => $order_currency,
+							'quantity' => 1,
+							'price'    => $fee->get_amount( false ),
+							'fee'      => $fee->get_amount( false ),
+							'total'    => $fee->get_amount( false ),
```

Comment:
Why `false`? In theory this isn't a valid context:
https://github.com/woocommerce/woocommerce/blob/bb244484d38768bc6c33a9423f0a5d5db75976f4/plugins/woocommerce/includes/class-wc-order-item-fee.php#L200